utils/src/main/scala/cromwell/pipeline/utils/configs/LogInfoUtils.scala

```diff
@@ -0,0 +1,84 @@
+package cromwell.pipeline.utils.configs
+
+import cromwell.pipeline.utils._
+import cromwell.pipeline.utils.configs.SecretData._
+import play.api.libs.functional.syntax.{ toFunctionalBuilderOps, unlift }
+import play.api.libs.json.Format.GenericFormat
+import play.api.libs.json.{ Json, Writes, __ }
+
+trait LogInfoUtils {
+  def getConfigMessage: String
+}
+
+object LogInfoUtils {
```

Comment:
How this object is related to the Logging?

Reply:
Something like "ConfigsToStringUtils" would definitely be a better option, thanks!

Reply:
Having `Utils` at the end still looks very much as Java naming.. :(
Please reconsider it.

Reply:
So that object is designed to collect configurations and make a nice readable string out of it. The name should encompass those properties, right?
How about: ConfigsPrinter, ConfigsToStringTransformer, ConfigsToString, ConfigsToJsonString, ConfigsMessenger, ConfigsMessageGenerator, ConfigsPrettyPrinter 
Any advice on proper naming would be really appreciated.

Reply:
Renamed to ConfigJsonOps